### DISCUSSION TITLE
Tab indexes

### DISCUSSION THREAD

**Discussion by bilal-fazlani** [squidfunk/mkdocs-material#5614] [Ideas]
Index page feature only exists for sections as now. 

https://squidfunk.github.io/mkdocs-material/setup/setting-up-navigation/#section-index-pages

It would be great to have it for tabs as well

**Comment by squidfunk** [comment]
If I understand you correctly, this is already supported. For example, if you go to our [blog](https://squidfunk.github.io/mkdocs-material/blog/), it directly links to `blog/index.md`. If that doesn't solve your problem, please provide more detail.

**Comment by bilal-fazlani** [comment]
So let's take https://squidfunk.github.io/mkdocs-material/reference/ page for instance.

<img width="1207" alt="image" src="https://github.com/squidfunk/mkdocs-material/assets/3396271/cc83db30-61b4-4c67-b42c-8551954051b5">

It looks like an index page to me.

There is some hierarchy visible here:

![image](https://github.com/squidfunk/mkdocs-material/assets/3396271/19a4d966-8ab4-4060-8fdd-4cc7ccda7aec)

But its only there because the index page does not have an icon. Once you add an icon, everything will look like its on the same level

![image](https://github.com/squidfunk/mkdocs-material/assets/3396271/1a2aaf09-b4d1-4a92-b99f-7ee3b5512872)

Once I remove the icon, I get the same effect as reference page

![image](https://github.com/squidfunk/mkdocs-material/assets/3396271/1d9f54be-d9b1-42e4-a26a-fd91bcdee585)

**Comment by bilal-fazlani** [comment]
This behaviour is a bit different from sections. When indexes are enabled, sections don't show up twice. 
But for tabs, it is appearing twice. Once in tab and again in side nav. At least on desktop view. 

That inconsistency makes me feel that indexes not working as a first class feature for tabs as it works for sections.

**Reply by squidfunk** [reply]
This is actually working as intended. While I understand that you or some other users might want to hide the title in the navigation, it makes sense to keep, because when you scroll down and don't have sticky tabs, the tabs will be gone, but the sidebar will still gice you the context in which section of the documentation you currently are. However, you can, of course, always hide it with CSS ☺️

**Reply by bilal-fazlani** [reply]
Thats a good idea. I will try that. Thanks! 😀
